File: JS Fundamentals/Arrays - Lab/day_of_week.js
function weekend(numb){
    let days = [
        'Monday',
        'Tuesday',
        'Wednesday',
        'Thursday',
        'Friday',
        'Saturday',
        'Sunday',
    ];
    let currentDay = 'Invalid day!';
    if(numb > 0 && numb < 8){
        currentDay = days[numb - 1];
    }
    console.log(currentDay)
}

weekend(3);
weekend(6);
weekend(11);

File: JavaFundamentals/src/Arrays/Lab/DayOfWeek.java
package Arrays.Lab;

import java.util.Scanner;

public class DayOfWeek {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int index = Integer.parseInt(scanner.nextLine());
        String[] days = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};

        if (index < 1 || index > days.length) {
            System.out.println("Invalid day!");
        } else {
            System.out.println(days[index - 1]);
        }
    }
}
